README.md

```diff
@@ -283,6 +284,24 @@ steps:
       acrUsername: ${{ secrets.REGISTRY_USERNAME }}
       acrPassword: ${{ secrets.REGISTRY_PASSWORD }}
 ```
+### Using dockerhub credentials to authenticate
+
+```yml
+steps:
+
+  - name: Log in to Azure
+    uses: azure/login@v1
+    with:
+      creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+  - name: Build and deploy Container App
+    uses: azure/container-apps-deploy-action@v1
+    with:
+      appSourcePath: ${{ github.workspace }}
+      registryUrl: mytestregistry.dockerhub.io
+      registryUsername: ${{ secrets.REGISTRY_USERNAME }}
+      registryPassword: ${{ secrets.REGISTRY_PASSWORD }}
+```
 
 This will create a new Container App named `github-action-container-app-<github-run-id>-<github-run-attempt>` in a new

```

Comment:
This description text should be under the above ACR example, and we should re-use this description text for the new example you've provided, but update the parts about it using ACR.